Dictionaries/eng_OCRFixReplaceList.xml

```diff
@@ -1195,9 +1195,11 @@
     <Word from="SoBl3lNG" to="SOBBING" />
     <Word from="Sojust" to="So just" />
     <Word from="soldierl" to="soldier!" />
-    <Word from="somethlng" to="somethlng" />
+    <Word from="somethlng" to="something" />
     <Word from="somethlng's" to="something's" />
     <Word from="somez'/7/ng" to="something" />
+    <Word from="somthing" to="something" />
++   <Word from="sumthing" to="something" />

```

Comment:
**You** are the admin of *diomed/subtitleedit*.  :smile: 
But, you can copy&paste [this corrected version](https://raw.githubusercontent.com/xylographe/subtitleedit/64b0c1cfeee70c441da1c088e9b76b6c87917266/Dictionaries/eng_OCRFixReplaceList.xml).